src/main/java/de/tum/in/www1/artemis/web/rest/dto/competency/LearningPathNavigationOverviewDTO.java

```diff
@@ -2,5 +2,5 @@
 
 import java.util.List;
 
-public record LearningPathNavigationOverviewDto(List<LearningPathNavigationObjectDTO> learningObjects) {
+public record LearningPathNavigationOverviewDTO(List<LearningPathNavigationObjectDTO> learningObjects) {
 }
```

Comment:
There's a failing architecture test: testDTOImplementations - can you check if this DTO needs a JsonInclude annotation?